withdraw-by-check/withdraw-by-check-json-contract.md

```diff
@@ -150,21 +152,21 @@ Unless otherwise noted, all values will be passed as double-quote encapsulated c
 **Error Code detail:**
 |Error Code| Logging Error Message Detail|
 |-|-|
-|500|Error Opening Letterfile BANNO.CHECK.WITHDRAW.V1.CFG: *[system generated error message]*
+|500|Error processing BANNO.CHECK.WITHDRAW.V1.CFG: *[system generated error message]*
 |501|Target Share/Loan available balance<=$0.00
-|501|Amount requested exceeds available
```

Comment:
Can you add back to the json contract
" - |501|Amount requested exceeds available"?
As I see that in the code

The rest of your changes look good!

Reply:
Hi Tom, Yes the PowerOn code in lines 220 - 222, still has these the following lines, so error code 501 can also mean that the, amount requested exceeds the available amount.

ELSE IF WDAMOUNT<=$0.00 OR
               WDAMOUNT>SLAVAILABLEBAL THEN
        DO
         ERRORCODE="501"
         ERRORMSG="loggingErrorMessage"
         TMPCHR=FORMAT(" ###,##9.99",WDAMOUNT)
         CALL NLS
         ERRORDETAIL="Amount req. "+TMPCHR
         TMPCHR=FORMAT(" ###,##9.99",SLAVAILABLEBAL)
         CALL NLS
         ERRORDETAIL=ERRORDETAIL+" exceeds avail. "+TMPCHR
         CALL ERRORHANDLER
        END

and your previous json, in line 155, included that description, but it is gone now
|501|Target Share/Loan available balance<=$0.00
|501|Amount requested exceeds available